golang-regex-validator/Readme.md

```diff
@@ -0,0 +1,14 @@
+
+# Golang Regex Validator
+
+This tool is aimed to validate Golang Regex expressions.
```

Comment:
How about being more descriptive here, e.g. like this?

```suggestion
This tool reads a Golang regular expression from the standard input and checks whether it can be compiled. The result is indicated by the exit status.
```

Reply:
Sorry, I renamed the file before applying the suggestion. Added manually.